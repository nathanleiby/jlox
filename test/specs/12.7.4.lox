class Foo {
  init() {
    print "this is ok";
    return;
  }
}

var f = Foo();
print f;
