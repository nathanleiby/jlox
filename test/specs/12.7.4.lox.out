this is ok
Foo instance
